cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -275,6 +275,13 @@ impl Contains<Location> for AmbassadorEntities {
 	}
 }
 
+pub struct SecretaryEntities;
+impl Contains<Location> for SecretaryEntities {
+	fn contains(location: &Location) -> bool {
+		matches!(location.unpack(), (1, [Parachain(1001), PalletInstance(93)]))
```

Comment:
I'd put both 1001 and 93 into constants describing what they are, `COLLECTIVES_PARA_ID` and `COLLECTIVES_SECRETARY_SALARY_PALLET_ID` it looks like

Reply:
Thanks for the review!

This has been update.